html/changelogs/example.yml

```diff
@@ -33,5 +33,12 @@ delete-after: True
 # Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
 # Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
 changes: 
-  - rscadd: "Added a changelog editing system that should cause fewer conflicts and more accurate timestamps."
-  - rscdel: "Killed innocent kittens."
+  - bugfix: "Fixed and overhauled diona light mechanics. Diona will survive comfortably in darkness for two minutes, suffer and lose health for a farther two minutes, and then spend a minute lying helplessly until death. Nymphs last 20% longer"
+  - tweak: "Diona nymphs can no just evolve into a gestalt after waiting a short amount of time. They must now eat things and accumulate a stockpile of biomass in order to evolve."
+  - rscadd: "Added a devouring system to allow mobs to eat other mobs and gain nutrition from it, either by swallowing them whole (if small enough), or eating them piece by piece if larger."
+  - rscadd: "Diona Nymphs can use devouring to eat organic (non-humanoid, non-synthetic, non-supernatural) mobs"
+  - rscadd: "Diona nymphs can now process chemicals properly, and consume normal food items (meals, fruit, meat, etc)"
+  - rscadd: "Diona nymphs can now harvest fully grown plants from botany, and can eat dead plants and weeds. Require tray lids to be open"
+  - rscadd: "Nymphs and other small creatures can now gnaw open cardboard boxes to get to their contents"
+  - tweak: "Flashlights can no longer be stacked. Only one held flashlight at a time will emit light"
+  - tweak: "Diona are affected far less by flashlights. They should find other sources of light to sustain themselves"

```

Comment:
made a couple of attempts, can't seem to get this back to basic state. Ill fix it by leaving it out next time i do a merge-rebase